<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YUV_Viewer</class>
 <widget class="QMainWindow" name="YUV_Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>690</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1180</width>
    <height>690</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="2" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>226</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" rowspan="3">
     <widget class="QGroupBox" name="image_box">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>yuv image preview</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="image_layout"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>420</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>yuv image parameter</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0" rowspan="2" colspan="2">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>SIZE       :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Width     :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="imwidth">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Height    :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="imheight">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>10</pointsize>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Bit  Depth :</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QSpinBox" name="spin_bit">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="minimum">
              <number>8</number>
             </property>
             <property name="maximum">
              <number>16</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>YUV Format :</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QComboBox" name="yuv_format">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <item>
              <property name="text">
               <string>YUV444</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>YUV422</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>YUV420</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="size_combox">
             <item>
              <property name="text">
               <string>&lt;Custom Size&gt;</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>VGA (640x480)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>WVGA (832x480)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ITU (720x576)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>720P (1280x720)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1080P (1920x1080)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4K (3840x2160)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>XGA (1024x768)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>XGA+ (1280x960)</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="1">
             <widget class="QCheckBox" name="planar_check">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>11</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Planar</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="packed_check">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>11</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Packed</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>10</pointsize>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Conversion   :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="conversion_combx">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <item>
              <property name="text">
               <string>BT2020</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>BT601</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Rec709</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>10</pointsize>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Components :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="component_combx">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <item>
              <property name="text">
               <string>YCbCr</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Y</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Cb</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Cr</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btn_open">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Open</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="save">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
